src/utils.js

```diff
@@ -7,3 +8,27 @@ export function debounce(fn, t) {
     }, t);
   };
 }
+
+/**
+ * Convert a number 0-127 to a set of days of week
+ * @param {Number} num
+ * @returns a list of days of week
+ */
+export function numberToDaysOfWeek(num) {
```

Comment:
Check below, if you are confused about this when adding job @nehakumari2307 and displaying job @kendebacker 
![image](https://user-images.githubusercontent.com/13199342/225788271-7e38f862-fe55-4a6c-b3cd-7582bbfb1536.png)